.github/workflows/cleanup-partner-data.yml

```
@@ -0,0 +1,118 @@
+            --port "${{ github.event.inputs.db_port }}" \
+            --user "${{ github.event.inputs.db_user }}"
+
+      - name: Disconnect Wireguard
+        if: ${{ always() && github.event.inputs.enable_wireguard == 'true' }}
+        run: |
+          sudo wg-quick down wg0 || true
+          echo "Wireguard VPN disconnected"
+
+      - name: Summary
+        if: success()
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:11:66: Double quote to prevent globbing and word splitting

(shellcheck)